README.md

```diff
@@ -145,41 +163,52 @@ $appConfig = [
     ]
 ];
 
+$container = new Container();
+Configurator::apply()->configFromArray($config)->to($container);
+
+$logger = $container->get('logger'));
 ```
 
 ### Configuring Inflectors
 
+**Currently only supported by the League Container.**
+
 It is also possible to set up
 [Inflectors](http://container.thephpleague.com/inflectors/) by adding an
-`inflectors` key to the config.
+`inflectors` key to the `di` section of the config.
 
 ```php
 $appConfig = [
-    'inflectors' => [
-        LoggerAwareInterface::class => [
-            'setLogger' => ['Some\Logger']
-        ]
-    ]
+    'di' => [
+        'inflectors' => [
+            LoggerAwareInterface::class => [
+                'setLogger' => ['Some\Logger']
+            ],
+        ],
+    ],
 ];
 ```
 
 ### Extra Settings
 
-You can provide an array of extra settings as a second parameter to
-`TomPHP\ConfigServiceProvider\ConfigServiceProvider::fromConfig()`.
-
-Current valid keys are:
-
-| Name        | Effect                                        |
-|-------------|-----------------------------------------------|
-| `prefix`    | Changes `config` prefix given to config keys. |
-| `separator` | Changes `.` separator in config keys.         |
-
-Example:
+Both `ConfigureContainer::fromArray($container, array $config, array $settings = [])`
+and `ConfigureContainer::fromFiles($container, array $files, array $settings = [])`
+can take an optional array of settings are the third parameter.
```

Comment:
@tomphp 

Looks like we need to update this paragraph to explain the use of `withSetting()`, right?